### DISCUSSION TITLE
Win11x64 PermissionError ELM 327 USB

### DISCUSSION THREAD

**Discussion by MisterAssm** [cedricp/ddt4all#1206] [Q&A] [answered]
Bonjour,
Je bloque depuis un petit moment sur l'application, lorsque j'appuie sur Tests ELM, j'ai une PermissionError (malgré le fait que j'essaye d'ouvrir DDT en administrateur) ou bien OSError parce qu'il ne trouve pas l'ELM.
L'ELM est pourtant fonctionnel avec d'autres applications (EOBD). J'ai essayé plusieurs ports.

Si quelqu'un aurait une idée d'où pourrait venir l'erreur..

![image](https://github.com/user-attachments/assets/18cd9b85-79f4-4006-8912-c8489b31556c)

**Comment by Furtif** [comment]
vous êtes sur que c'est le port com OBD ??? Ce error arrive si le port est en use ....

**Comment by MisterAssm** [comment]
> vous êtes sur que c'est le port com OBD ??? Ce error arrive si le port est en use .... 

Je pense, du moins le port fonctionne sans problème avec d'autres applications (EOBD). J'avais essayé dans le doute avec les deux ports de mon pc mais même problème.

**Reply by Furtif** [reply]
```d'autres applications (EOBD).``` ne veut pas dire que DDT4ALL utilise mêmes choses.
Par exemple mon premier achat était un fail total, je avais acheter un ODB pour des applications mercedes/Iveco

**Reply by Furtif** [reply]
du coup https://github.com/cedricp/ddt4all/issues/1031 j'ai su répondre avec précision a ce post.  https://github.com/cedricp/ddt4all/discussions/1032

**Comment by Furtif** [comment]
> > vous êtes sur que c'est le port com OBD ??? Ce error arrive si le port est en use ....
> 
> Je pense, du moins le port fonctionne sans problème avec d'autres applications (EOBD). J'avais essayé dans le doute avec les deux ports de mon pc mais même problème.

oui possible, mais ddt4all ne fonctionne pas avec tous OBD il faudra utiliser l'un de ceux mentionné dans readme https://github.com/cedricp/ddt4all?tab=readme-ov-file#supported-diagnostic-adapters-so-far

j'ai 2 OBD un wifi l'autre USB et les 2 fonctionnent bien sous win11.

**Comment by MisterAssm** [comment]
> > > vous êtes sur que c'est le port com OBD ??? Ce error arrive si le port est en use ....
> > 
> > Je pense, du moins le port fonctionne sans problème avec d'autres applications (EOBD). J'avais essayé dans le doute avec les deux ports de mon pc mais même problème.
> 
> oui possible, mais ddt4all ne fonctionne pas avec tous OBD il faudra utiliser l'un de ceux mentionné dans readme https://github.com/cedricp/ddt4all?tab=readme-ov-file#supported-diagnostic-adapters-so-far
> 
> j'ai 2 OBD un wifi l'autre USB et les 2 fonctionnent bien sous win11.

En voulant vérifier, la page sur laquelle j'ai commandé mon elm n'est plus disponible.. J'en recommande un autre, c'est peut être ça le problème. Je vous tiens au courant.

**Comment by Furtif** [comment]
> > > > vous êtes sur que c'est le port com OBD ??? Ce error arrive si le port est en use ....
> > > 
> > > 
> > > Je pense, du moins le port fonctionne sans problème avec d'autres applications (EOBD). J'avais essayé dans le doute avec les deux ports de mon pc mais même problème.
> > 
> > 
> > oui possible, mais ddt4all ne fonctionne pas avec tous OBD il faudra utiliser l'un de ceux mentionné dans readme https://github.com/cedricp/ddt4all?tab=readme-ov-file#supported-diagnostic-adapters-so-far
> > j'ai 2 OBD un wifi l'autre USB et les 2 fonctionnent bien sous win11.
> 
> En voulant vérifier, la page sur laquelle j'ai commandé mon elm n'est plus disponible.. J'en recommande un autre, c'est peut être ça le problème. Je vous tiens au courant.

OK ok 👍

**Comment by Furtif** [comment]
> > > > > vous êtes sur que c'est le port com OBD ??? Ce error arrive si le port est en use ....
> > > > 
> > > > 
> > > > Je pense, du moins le port fonctionne sans problème avec d'autres applications (EOBD). J'avais essayé dans le doute avec les deux ports de mon pc mais même problème.
> > > 
> > > 
> > > oui possible, mais ddt4all ne fonctionne pas avec tous OBD il faudra utiliser l'un de ceux mentionné dans readme https://github.com/cedricp/ddt4all?tab=readme-ov-file#supported-diagnostic-adapters-so-far
> > > j'ai 2 OBD un wifi l'autre USB et les 2 fonctionnent bien sous win11.
> > 
> > 
> > En voulant vérifier, la page sur laquelle j'ai commandé mon elm n'est plus disponible.. J'en recommande un autre, c'est peut être ça le problème. Je vous tiens au courant.

Par contre soit sur qu'il a un chip PIC18F25K80 comme décrit dans readme !!!!

**Comment by Furtif** [comment]
[ODB PIC18F25K80](https://www.amazon.fr/Diagnostic-Voiture-Danalyse-PIC18F25K80-Protocoles/dp/B0C6H5HD2T/ref=sr_1_1_sspa?dib=eyJ2IjoiMSJ9.QxuazZCAtri4JIqhZC-ePPkRT1gu98Q2d0skkP4DS2IOuBF2yWxfG_9TIq5XN5hOqFGDTKgcOF45kMdQ6iLdOjJQmFL4KZA_0sojLAff_6lVPFUjNu7GNF9VEukhZ-bTyf5QpQRzua8eq3uLnPyv47ZmSA7E4lZQdStzjp-kZT5AmWxD4Tn7xkFaEHo3NKpncdPjho26IN-K6GRYgqbzjL5MHb7PaLXpETXAPc8rU66iBCznhVEc62YwX9maBN-DB43Y7yrEjn9xHVg-FIf9ud7gx_2yNZqBdXG4BZzZqJY.DkGxYlMm-Wk1GmpuLyP44NaFVJcJTcgbRoSk4T1VE4k&dib_tag=se&keywords=pic18f25k80&qid=1724337857&sr=8-1-spons&sp_csd=d2lkZ2V0TmFtZT1zcF9hdGY&psc=1) 

type celui la.

**Reply by pedja76** [reply]
> [ODB PIC18F25K80](https://www.amazon.fr/Diagnostic-Voiture-Danalyse-PIC18F25K80-Protocoles/dp/B0C6H5HD2T/ref=sr_1_1_sspa?dib=eyJ2IjoiMSJ9.QxuazZCAtri4JIqhZC-ePPkRT1gu98Q2d0skkP4DS2IOuBF2yWxfG_9TIq5XN5hOqFGDTKgcOF45kMdQ6iLdOjJQmFL4KZA_0sojLAff_6lVPFUjNu7GNF9VEukhZ-bTyf5QpQRzua8eq3uLnPyv47ZmSA7E4lZQdStzjp-kZT5AmWxD4Tn7xkFaEHo3NKpncdPjho26IN-K6GRYgqbzjL5MHb7PaLXpETXAPc8rU66iBCznhVEc62YwX9maBN-DB43Y7yrEjn9xHVg-FIf9ud7gx_2yNZqBdXG4BZzZqJY.DkGxYlMm-Wk1GmpuLyP44NaFVJcJTcgbRoSk4T1VE4k&dib_tag=se&keywords=pic18f25k80&qid=1724337857&sr=8-1-spons&sp_csd=d2lkZ2V0TmFtZT1zcF9hdGY&psc=1)
> 
> Ukцajte ovaj.

My respect.
Would you clarify with this model when the MS CAN bus is used, and in which cases the HS CAN bus.
The car I would use is SCENIK III (changes to the main dashboard and multimedia.)

**Reply by AndyIreland** [reply]
you dont need HS CAN modification for Dashboard , but you would for Multimedia (on earlier Renault Cars without Security Gateway - later renaults have Security gateway and in that case dont have HS / CAN2 /CAN-HI to access the multimedia. Thats my understanding anyway.

**Reply by pedja76** [reply]
Hvala na odgovoru.
:)

**Comment by Furtif** [comment]
Ou sa https://github.com/cedricp/ddt4all/discussions/1205#discussioncomment-10421650

**Comment by Furtif** [comment]
vais changer ceci vers discussions car il n'est pas tout a fait un issue, si était le cas il aurais plusieurs users avec le même problème.

**Comment by MisterAssm** [comment, marked as answer]
Effectivement, la source de mes problèmes était mon ancien ELM. Je n'ai rencontré aucun soucis avec le nouveau, merci beaucoup pour les renseignements.

**Reply by Furtif** [reply]
Parfait
